docs/operations/karpenter.md

```diff
@@ -2,68 +2,117 @@
 
 [Karpenter](https://karpenter.sh) is a Kubernetes-native capacity manager that directly provisions Nodes and underlying instances based on Pod requirements. On AWS, kOps supports managing an InstanceGroup with either Karpenter or an AWS Auto Scaling Group (ASG).
 
+## Prerequisites
+
+Managed Karpenter requires kOps 1.34+ and that [IAM Roles for Service Accounts (IRSA)](/cluster_spec#service-account-issuer-discovery-and-aws-iam-roles-for-service-accounts-irsa) be enabled for the cluster.
+
 ## Installing
 
-If using kOps 1.26 or older, enable the Karpenter feature flag :
+### New clusters
 
 ```sh
-export KOPS_FEATURE_FLAGS="Karpenter"
+export NAME="my-cluster.example.com"
+export REGION="us-east-1"
+export ZONE="us-east-1a"
+
+kops create cluster --name ${NAME} \
+  --state=s3://my-state-store \
+  --discovery-store=s3://my-discovery-store \
+  --cloud=aws \
+  --networking=cilium \
+  --zones=${ZONE} \
+  --instance-manager=karpenter \
+  --yes
 ```
 
-Karpenter requires that external permissions for ServiceAccounts be enabled for the cluster. See [AWS IAM roles for ServiceAccounts documentation](/cluster_spec#service-account-issuer-discovery-and-aws-iam-roles-for-service-accounts-irsa) for how to enable this. 
-
 ### Existing clusters
 
-On existing clusters, you can create a Karpenter InstanceGroup by adding the following to its InstanceGroup spec:
+The Karpenter addon must be enabled in the cluster spec:
 
 ```yaml
 spec:
-  manager: Karpenter
+  karpenter:
+    enabled: true
 ```
 
-You also need to enable the Karpenter addon in the cluster spec:
+To create a Karpenter InstanceGroup, set the following in its InstanceGroup spec:
 
 ```yaml
 spec:
-  karpenter:
-    enabled: true
+  manager: Karpenter
 ```
 
-### New clusters
-
-On new clusters, you can simply add the `--instance-manager=karpenter` flag:
+### EC2NodeClass and NodePool
 
 ```sh
-kops create cluster --name mycluster.example.com --cloud aws --networking=amazonvpc --zones=eu-central-1a,eu-central-1b --master-count=3 --yes --discovery-store=s3://discovery-store/
+export USER_DATA=$(aws s3 cp s3://my-state-store/${NAME}/igconfig/node/nodes/nodeupscript.sh -)
```

Comment:
A toolbox command would be nice here IMO (but can be a future PR).  Could also be functionality of `kops create ig`, but should probably start as a toolbox command